sentry-core/src/hub.rs

```diff
@@ -284,23 +286,20 @@ impl Hub {
         }}
     }
 
+    /// Capture any `std::error::Error`.
+    pub fn capture_error<E: Error + ?Sized>(&self, error: &E) -> Uuid {
+        let event = event_from_error(error);
+        self.capture_event(event)
+    }
```

Comment:
Can we move this into `error.rs`, to keep all the error-related functions together? That way the file would look similar to integrations.

Reply:
Can we guard this in a hub call or client to avoid work if the client is disabled? I'm realizing that this also applies to `capture_anyhow` and `capture_fail`.